### PR TITLE
Listed the added status and corrected the json

### PR DESCRIPTION
**Pull request by ShahinaKP**
@nishintn 

Corrected the json file and combined to single json
Listed the added status to the StatusHistoryComponent component

Please review and merge the branch

### CODE REVIEW

File: src/app/assets/json/statusInput.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+	"projects" :
+		[
+			"N/A",
+			"Project 1",
```
Comment (mad-nishin) [inline src/app/assets/json/statusInput.json]: Please use meaningful data set.

File: src/app/components/AddDailyStatusComponent.jsx [hunk 2]
```diff
@@ -6,6 +6,9 @@ class AddDailyStatusComponent extends React.Component {
   constructor(props) {
     super(props);
 
+    this.selHr = "01";
```
Comment (mad-nishin) [inline src/app/components/AddDailyStatusComponent.jsx]: What is this used for?

File: src/app/components/AddDailyStatusComponent.jsx [hunk 3]
```diff
@@ -69,23 +102,23 @@ class AddDailyStatusComponent extends React.Component {
           <div className="colWrapper">
             <div className="col2">
               <label>Date</label>
-              <select>{getOptions(this.state.dateOptions)}</select>
+              <select value={this.state.selDate} onChange={this.onChangeHandler.bind(this, "date")}>{getOptions(this.state.dateOptions)}</select>
```
Comment (mad-nishin) [inline src/app/components/AddDailyStatusComponent.jsx]: What is bind used for?

**Comment by mad-nishin** [comment]
@ShahinaKP ,
Please address my comments.

File: src/app/assets/json/statusInput.json (hunk 1 above)
Comment (ShahinaKP) [reply src/app/assets/json/statusInput.json]: Corrected.  Updated with proper project names.

File: src/app/components/AddDailyStatusComponent.jsx (hunk 2 above)
Comment (ShahinaKP) [reply src/app/components/AddDailyStatusComponent.jsx]: Used to initialise the value but now this property moved to state

File: src/app/components/AddDailyStatusComponent.jsx (hunk 3 above)
Comment (ShahinaKP) [reply src/app/components/AddDailyStatusComponent.jsx]: This binding is necessary to make `this` work in the callback. Now the binding also move to the constructor. All events are binded inside the constructor like:
this.onChangeHandler = this.onChangeHandler.bind(this);
this.onAddActivity = this.onAddActivity.bind(this);

**Comment by ShahinaKP** [comment]
@nishintn 

Changed files based on review comments
Added description field
Completed the functionalities.

Please review and merge the branch

**Pull request merged**